application-mocca-calendar-api/src/main/java/org/xwiki/contrib/moccacalendar/internal/utils/DefaultEventAssembly.java

```diff
@@ -112,15 +123,21 @@ public List<DocumentReference> executeQuery(EventQuery query) throws QueryExcept
      * It will be removed and replaced by a throughout use of the "viewable" query filter.
      *
      * @param eventDocRefs a list of strings, not null.
+     * @param wiki wiki identifier
      * @return the corresponding documents, filtered by view rights.
      */
-    public List<DocumentReference> filterViewableEvents(List<String> eventDocRefs)
+    public List<DocumentReference> filterViewableEvents(List<String> eventDocRefs, String wiki)
     {
         List<DocumentReference> visibleRefs = new ArrayList<>();
         // check view rights on results ... should use "viewable" filter when minimal platform version is >= 9.8
```

Comment:
The minimal XWiki version is 11.10 so you should drop this code (both ``filterViewableEvents`` signatures) and use the "viewable" filter instead.

Reply:
Right, the latest commit usses the "viewable" query filter indeed.